flags.go

```diff
@@ -208,13 +208,13 @@ func newApp() (app *cli.App) {
 			cli.DurationFlag{
 				Name:  "stat-cache-ttl",
 				Value: time.Minute,
-				Usage: "How long to cache StatObject results and inode attributes.",
+				Usage: "How long to cache StatObject results and inode attributes. It is on its way to be deprecated in the next few releases and will be replaced with metadata-config:ttl-secs in the gcsfuse config-file. For now, for backward compatibility, a minimum of stat-cache-ttl and type-cache-ttl values are taken, the result is rounded up to the next higher multiple of seconds and that is used as ttl for both stat-cache and type-cache.",
```

Comment:
@sethiay PTAL at the updated description here.

Reply:
We can shorted this description.
Note: This flag will be deprecated in future and only with metadata-config:ttl-secs in the gcsfuse config-file will be supported. For now, a minimum of stat-cache-ttl and type-cache-ttl values, rounded up to the next higher multiple of seconds is used as ttl for both stat-cache and type-cache.

Reply:
Updated as suggested.